src/jmcomic/jm_config.py

```
@@ -565,8 +608,8 @@ def register_exception_listener(cls, etype, listener):
 disable_jm_log = JmModuleConfig.disable_jm_log
 
 
-class PrettyFormatter(logging.Formatter):
-    """带 ANSI 颜色的日志格式化器，按 topic 前缀分配颜色"""
+class PrettyFormatter(JmLogFormatter):
+    """带 ANSI 颜色的日志格式化器，按 topic 前缀分配颜色。"""
```

Ruff (0.16.0)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)